lib/commands/location.js

```diff
@@ -35,8 +35,11 @@ commands.setGeoLocation = async function (location) {
   }
   const args = ['-u', this.opts.udid];
   args.push(...formatLocationArg(latitude));
-  args.push(...formatLocationArg(longitude));
-  try {
+  if (latitude.startsWith('-')) {
+    args.push(...(longitude));
+  } else {
+    args.push(...formatLocationArg(longitude));
```

Comment:
this will probably won't work, because idevicelocation requires double dash to be put before both arguments if both of them are negative:

```
-- -77.432332 -7.008373564
```